reconcile/aws_account_manager/merge_request_manager.py

```diff
@@ -88,10 +88,8 @@ def create_account_file(
             self._vcs.get_file_content_from_app_interface_master(
                 file_path=account_tmpl_file_path
             )
-            # File already exists
-            raise FileExistsError(
-                f"File {account_tmpl_file_path} already exists in the repository"
-            )
+            # File already exists. nothing to do.
```

Comment:
nit: I think it wouldn't hurt to log an debu/info explanatory message. Empty returns are always hard to analyze for the non-initiated, I've been there a few times

Reply:
You're right. Applied in [22f6945](https://github.com/app-sre/qontract-reconcile/pull/4707/commits/22f69452ce88af1b45dd6aa39a8d67e33372d9f7)